pkg/operator/controller/machineconfig/mcp_test.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package machineconfigcontroller
 
+/*
```

Comment:
What about those?

Reply:
Since there are new changes added, thought to make use of unit test story to add new cases and existing cases also require changes, so commented those, to rework everything at once.